internal/logging/logging.go

```diff
@@ -238,7 +238,7 @@ func writeMessageDepth(depth int, level string, msg string, args ...interface{})
 			}
 			errMsg += ": " + errw.Error()
 		}
-		fmt.Fprintf(os.Stderr, "Error writing log message: %s\n", errMsg)
+		fmt.Fprintf(os.Stderr, "Error writing log message, please contact support: %s\n", errMsg)
```

Comment:
Nit: We may want to provide the user with an email or website where they can contact support.